extensions/ipynb/src/serializers.ts

```diff
@@ -4,9 +4,11 @@
  *--------------------------------------------------------------------------------------------*/
 
 import type * as nbformat from '@jupyterlab/nbformat';
+import { TextDecoder } from 'util';
```

Comment:
I think we don't need this import, not sure if having it explicit would break the browser build.